content/blog/2024-05-20-summer-software-open-science-school.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Applications now open for a Summer School in Open Science + Research Software Engineering"
+date: 2024-05-20
+author: "Madicken Munk"
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
+
+This workshop complements previous summer and winter schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering.  
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+
+## Target Audience
```

Comment:
perhaps say something about being open to all disciplines (if this is right), including humanities and areas that aren't considered "science"?

Reply:
Or maybe this is just for "science"?

Reply:
Thank you so much for pointing this out! I added language saying that all domains are welcome and that we are targeting people who use or develop software in their research.